iree/compiler/Codegen/Common/LinalgBufferizePass.cpp

```diff
@@ -472,9 +478,12 @@ static SmallVector<Value, 4> getAliasingBuffersForResults(
             tensor::CastOp>([&](auto singleResultOp) -> SmallVector<Value, 4> {
         return {getAliasingBufferForResult(b, singleResultOp, bvm)};
       })
-      .Case<linalg::TensorCollapseShapeOp, linalg::TensorExpandShapeOp>(
-          [&](auto reshapeOp) -> SmallVector<Value, 4> {
-            return {getAliasingBufferForReshapeResult(b, reshapeOp, bvm)};
+      .Case<linalg::TensorExpandShapeOp>([&](auto op) -> SmallVector<Value, 4> {
```

Comment:
If you rename the methods `getAliasingBufferForResult`, you can just add `linalg::TensorExpandShapeOp` and `linalg::TensorCollapseShapeOp` to the list at line 477.